examples/realworld/ext/article/components/ArticleViewPage.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import ReactMarkdown from 'react-markdown'
+import moment from 'moment'
+import PropTypes from 'prop-types'
+
+import useAuth from '@wasp/auth/useAuth.js'
+import { useQuery } from '@wasp/queries'
+
+import getArticle from '@wasp/queries/getArticle'
+import getArticleComments from '@wasp/queries/getArticleComments'
+import getUser from '@wasp/queries/getUser'
+import deleteArticle from '@wasp/actions/deleteArticle'
+import createComment from '@wasp/actions/createComment'
+import deleteComment from '@wasp/actions/deleteComment'
```

Comment:
might be cool if we could also group actions somehow, so you don't have to import them all separately here, but more like `import { delete, create, update } from '@wasp/actions/article'`

Reply:
True! I also thought about that. I added that thought into README.md.